atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -32,7 +32,7 @@ def add_yum_repos_to_dockerfile(yumrepos, df, inherited_user):
     # Insert the ADD line at the beginning of each stage
     df.add_lines(
         "ADD %s* '%s'" % (RELATIVE_REPOS_PATH, YUM_REPOS_DIR),
-        all_stages=True, at_start=True
+        all_stages=True, at_start=True, skip_scratch=True
```

Comment:
Should bump the minimum version of dockerfile-parse listed in requirements.txt.

Reply:
updated in atomic_reactor.spec as well

Reply:
ok